includes/class-sensei-grading.php

```diff
@@ -1156,6 +1156,40 @@ public function get_graded_lessons_average_grade() {
 		return $average_grade_value;
 	}
 
+	/**
+	 * Get average grade of all lessons graded in all the courses filtered by students.
+	 *
+	 * @since 4.5.0
+	 * @access public
+	 *
+	 * @param array $user_ids user ids.
+	 * @return double $graded_lesson_average_grade Average value of all the graded lessons in all the courses.
+	 */
+	public function get_graded_lessons_average_grade_filter_users( $user_ids ) {
```

Comment:
I think you should do the same refactor as with `get_average_days_to_completion` - move this method to `Sensei_Reports_Overview_List_Table_Students` or `Sensei_Reports_Overview_Service_Students` and deprecate the old one.

Reply:
Hey Miro since the old function is not jet in the **trunk** can we maybe just remove it?

Reply:
Moved it to the new class  Sensei_Reports_Overview_Service_Students https://github.com/Automattic/sensei/pull/5091/commits/dc828be6b2dd1d535e35aa706979ee56e432ae4e

Reply:
One question and the same goes for the comments above, the methods we are deprecating here are added in the same version they are going to be deprecated in 4.5.0. Since we still haven't merged the changes to the **trunk** branch. 
Do you know if we can just remove them?

Reply:
Oh, sorry, maybe I didn't explain it right, here I meant to deprecate `Sensei_Grading::get_graded_lessons_average_grade` and move `Sensei_Grading::get_graded_lessons_average_grade_filter_users`.